src/middlewared/middlewared/plugins/iscsi_/extents.py

```diff
@@ -68,26 +68,7 @@ async def sharing_task_determine_locked(self, data, locked_datasets):
 
         return await self.middleware.call('pool.dataset.path_in_locked_datasets', path, locked_datasets)
 
-    @accepts(Dict(
-        'iscsi_extent_create',
-        Str('name', required=True, max_length=MAX_EXTENT_NAME_LEN),
-        Str('type', enum=['DISK', 'FILE'], default='DISK'),
-        Str('disk', default=None, null=True),
-        Str('serial', default=None, null=True),
-        Str('path', default=None, null=True),
-        Int('filesize', default=0),
-        Int('blocksize', enum=[512, 1024, 2048, 4096], default=512),
-        Bool('pblocksize'),
-        Int('avail_threshold', validators=[Range(min_=1, max_=99)], null=True),
-        Str('comment'),
-        Bool('insecure_tpc', default=True),
-        Bool('xen'),
-        Str('rpm', enum=['UNKNOWN', 'SSD', '5400', '7200', '10000', '15000'],
-            default='SSD'),
-        Bool('ro', default=False),
-        Bool('enabled', default=True),
-        register=True
-    ), audit='Create iSCSI extent', audit_extended=lambda data: data["name"])
+    @api_method(IscsiExtentCreateArgs, IscsiExtentCreateResult, audit='Create iSCSI extent', audit_extended=lambda data: data['name'])
```

Comment:
flake8 here (line too long)

Reply:
Addressed.

But FMI what is the max length permitted by your flake8?

Reply:
120